lib/extends/exporters/proposal_serializer.rb

```diff
@@ -50,18 +49,37 @@ def serialize
       }
     }
   end
-  # rubocop:enable Metrics/CyclomaticComplexity
 
   protected
 
   def i18n_scope
     "decidim.proposals.admin.exports.column_name.proposal"
   end
 
+  def creator_username
+    return "" if proposal.creator_identity.blank?
+
+    proposal.creator_identity.try(:user_name).presence || proposal.creator_identity.try(:name).presence || ""
+  end
+
+  def creator_email
+    return "" if proposal.creator_identity.blank?
+
+    proposal.creator_identity.try(:email).presence || ""
+  end
+
+  def creator_phone_number
+    return "" if proposal.creator_identity.blank?
+
+    phone_number(proposal.creator_identity.try(:user_id)) || ""
+  end
+
   # phone_number retrieve the phone number of an user stored from phone_authorization_handler
   # Param: user_id : Integer
   # Return string, empty or with the phone number
   def phone_number(user_id)
+    return "" if user_id.blank?
```

Comment:
If we add condition in `create_phon_number`, this guard clause is no longer needed